src/test/operators/export_binary_test.cpp

```diff
@@ -164,6 +164,27 @@ TEST_F(OperatorsExportBinaryTest, FixedStringDictionarySegment) {
   EXPECT_TRUE(compare_files("resources/test_data/bin/StringDictionarySegment.bin", filename));
 }
 
+TEST_F(OperatorsExportBinaryTest, StringRunLengthSegment) {
+  TableColumnDefinitions column_definitions;
+  column_definitions.emplace_back("a", DataType::String, false);
+
+  auto table = std::make_shared<Table>(column_definitions, TableType::Data, 10);
+  table->append({"This"});
+  table->append({"is"});
```

Comment:
For run-length segments, it makes sense to have run-lengths > 1 in the tested data

Reply:
We now have a special multi-chunk test case with repeating values especially for RunLength-segments